app/services/album.js

```diff
@@ -8,10 +8,10 @@ exports.allAlbums = params =>
   axios
     .get(`${base_uri}/albums`)
     .then(response => {
```

Comment:
FYI (this is not a fix request, just an observation), you can use destructuring in the params: `.then(({ sortKey, sortOrder, filter }) => {...})`